contracts/TracerPerpetualSwaps.sol

```diff
@@ -464,7 +502,7 @@ contract TracerPerpetualSwaps is
         Balances.Position memory pos =
             Balances.Position(position.quote, position.base);
         uint256 minMargin =
-            Balances.minimumMargin(pos, price, gasCost, maxLeverage);
+            Balances.minimumMargin(pos, price, gasCost, trueMaxLeverage());
```

Comment:
Should all calcs such as `marginIsValid` and `userMarginIsValid` now use `trueMaxLeverage()` moving forward?